src/poll.rs

```diff
@@ -53,7 +119,32 @@ pub fn poll(fds: &mut [PollFd], timeout: libc::c_int) -> Result<libc::c_int> {
     Errno::result(res)
 }
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
+/// `ppoll()` allows an application to safely wait until either a file
```

Comment:
Let's try to shorten this a bit so it looks better in the docs. The man page starts with "wait for some event on a file descriptor". I think that's descriptive and succinct and I suggest you use that here instead.